### ISSUE TITLE
Setup a Map summarizing all results from a particular category

### ISSUE THREAD

**Issue by spMohanty** [geotagx/geotagx-theme#69]
The assumption being all the projects in the category are created using the geotagx_project_template

**Comment by spMohanty** [comment]
Has a dependency on https://github.com/geotagx/geotagx-theme/issues/62

**Comment by elliec** [comment]
@spMohanty should this be linked with HDX (#70)?

**Comment by spMohanty** [comment]
Deadline for the complete map : 20th August

**Comment by elliec** [comment]
@spMohanty calculate centroid of all polygons for each photo and add to GeoJSON, display only centroid but when users hovers (or clicks) or centroid it displays all polygons for that photo. Will need to be made seperate to HDX map, unless HDX allows this customisation

**Comment by elliec** [comment]
@spMohanty add basic drupal like filters for data

**Comment by elliec** [comment]
add filter to yemen project to filter for damage. or add filters for each question in a project if thats easier

**Comment by elliec** [comment]
highlight position marker and make polyogns stay until another marker is hovered over

**Comment by elliec** [comment]
@spMohanty could you please add the image source to the summary display in the map? and could you make the image smaller in the display? right now it is huge and takes up a lot of the screen (and i have a big screen), so smaller photos are not clear

**Comment by spMohanty** [comment]
@elliec : Added the max-width for the images in the summary. And also the source. Though, I have added the `image_url`(the actual link to the image) as the source instead of the `image_source`(the website where this image was found), as collecting the image_source at this end will be a computationally expensive operation for us when rendering the images. But if you think, we will definitely have to add the `image_source`, do let me know, and I ll replace the `image_url` with the `image_source`.

**Comment by elliec** [comment]
@spMohanty I think we will need the source but let me try it for a bit and see if i need it. I will let you know. Right now its broken though, the points arnt showing up (see email)

**Comment by spMohanty** [comment]
@elliec : Fixed the bug. and made it more tolerant to garbage coordinates. Let me know if this happens again, I ll also add some feedback messages for the user.

**Comment by elliec** [comment]
@spMohanty thanks looks heaps better. For now the image url is ok. I think i wanted it really to see where the photo was coming from, and you can get that from the source. I will let you know if i find the image source is necessary. 

We should probably add an explanation somewhere of what the points and polygons are, maybe just a link to the results page i was working on..

**Comment by spMohanty** [comment]
@elliec : Yeah sure !! If you have some text that you would want me to put somewhere on the page, let me know. I ll update it.

**Comment by elliec** [comment]
k, ill send you the text when its done, thanks!

**Comment by elliec** [comment]
@spMohanty could you add a link to the data explanation from the maps page? then we can close this

Thanks!
